backend-diagram-converter/webapp/src/main/java/org/camunda/community/converter/webapp/dto/ProcessDefinitionDiagramDto.java

```diff
@@ -0,0 +1,13 @@
+package org.camunda.community.converter.webapp.dto;
+
+public class ProcessDefinitionDiagramDto {
+  private String bpmn20Xml;
```

Comment:
Why do you use an Object as a wrapper around a String?

I don't see any value here.

Reply:
This is the structure the Rest API provides when fetching the XML of a process definition. I cannot change this...